src/main/java/net/minecraftforge/fml/ForgeI18n.java

```diff
@@ -88,10 +88,18 @@ public static void loadLanguageData(final Map<String, String> properties) {
 
     public static String parseMessage(final String i18nMessage, Object... args) {
         final String pattern = getPattern(i18nMessage);
-        return parseFormat(pattern, args);
+        try
+        {
+            return parseFormat(pattern, args);
+        }
+        catch (IllegalArgumentException e)
+        {
+            LOGGER.warn(CORE, "Invalid translation pattern for key {}!", i18nMessage);
+            return i18nMessage+" - invalid pattern";
```

Comment:
Do not add that suffix. Minecraft has never done that; before, it just returns the message.